evap/static/scss/components/_grid.scss

```diff
@@ -2,6 +2,16 @@
     background-color: $table-striped-bg;
 }
 
+.grid-row {
+    display: contents;
+}
```

Comment:
This seems a bit more global than I understand it to be, I wonder if we should have it nested in another selector, but there isn't really any particular one that comes to mind. Do you think that in any kind of grid layout (in the context of EvaP) we would want this display type on the row, or is it more incidentally needed here?

Reply:
I added the selector, so we can just rename it.

The issue is, that a grid does not allow nested elements, e.g. we have the table
```
A B C
D E F
```
Now we want the table to be striped, so DEF should get a gray background.
To do that, we need an element grouping the rows, so that we can select `:nth-child(2)`. but, with
```
<div>A B C</div>
<div>D E F</div>
```
the grid breaks.

The only solution is to apply `display: contents` to the wrapping element.

Therefore, in any table using CSS grid, the row-element should get `display: contents`